x-pack/plugins/observability_solution/synthetics/public/apps/embeddables/status_overview/status_overview_embeddable_factory.tsx

```diff
@@ -0,0 +1,103 @@
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import React, { useEffect } from 'react';
+import { DefaultEmbeddableApi, ReactEmbeddableFactory } from '@kbn/embeddable-plugin/public';
+import {
+  initializeTitles,
+  useBatchedPublishingSubjects,
+  fetch$,
+  PublishesWritablePanelTitle,
+  PublishesPanelTitle,
+  HasEditCapabilities,
+  SerializedTitles,
+} from '@kbn/presentation-publishing';
+import { BehaviorSubject, Subject } from 'rxjs';
+import type { StartServicesAccessor } from '@kbn/core-lifecycle-browser';
+import { SYNTHETICS_OVERVIEW_EMBEDDABLE } from '../constants';
+import { ClientPluginsStart } from '../../../plugin';
+import { StatusOverviewComponent } from './status_overview_component';
+
+export const getOverviewPanelTitle = () =>
+  i18n.translate('xpack.synthetics.statusOverview.displayName', {
+    defaultMessage: 'Synthetics Status Overview',
+  });
+
+export interface OverviewEmbeddableState extends SerializedTitles {
+  tags: string[];
+}
+
+export type StatusOverviewApi = DefaultEmbeddableApi<OverviewEmbeddableState> &
+  PublishesWritablePanelTitle &
+  PublishesPanelTitle &
+  HasEditCapabilities;
+
+export const getStatusOverviewEmbeddableFactory = (
+  getStartServices: StartServicesAccessor<ClientPluginsStart>
+) => {
+  const factory: ReactEmbeddableFactory<
+    OverviewEmbeddableState,
+    OverviewEmbeddableState,
+    StatusOverviewApi
+  > = {
+    type: SYNTHETICS_OVERVIEW_EMBEDDABLE,
+    deserializeState: (state) => {
+      return state.rawState as OverviewEmbeddableState;
+    },
+    buildEmbeddable: async (state, buildApi, uuid, parentApi) => {
+      const { titlesApi, titleComparators, serializeTitles } = initializeTitles(state);
+      const defaultTitle$ = new BehaviorSubject<string | undefined>(getOverviewPanelTitle());
+      const tags$ = new BehaviorSubject(state.tags);
+      const reload$ = new Subject<boolean>();
+
+      const api = buildApi(
+        {
+          ...titlesApi,
+          isEditingEnabled: () => false,
+          getTypeDisplayName: () => getOverviewPanelTitle(),
+          defaultPanelTitle: defaultTitle$,
+          onEdit: async () => {},
+          serializeState: () => {
+            return {
+              rawState: {
+                ...serializeTitles(),
+                tags: tags$.getValue(),
+              },
+            };
+          },
+        },
+        {
+          ...titleComparators,
+          tags: [tags$, (value) => tags$.next(value)],
+        }
+      );
+
+      const fetchSubscription = fetch$(api)
+        .pipe()
+        .subscribe((next) => {
+          reload$.next(next.isReload);
+        });
+
+      return {
+        api,
+        Component: () => {
+          const [] = useBatchedPublishingSubjects();
+
+          useEffect(() => {
+            return () => {
+              fetchSubscription.unsubscribe();
+            };
+          }, []);
+          return <StatusOverviewComponent reload$={reload$} />;
```

Comment:
@shahzad31 Every embeddable needs to have a `data-shared-item=""` in order to not have any timeout errors from the Kibana screenshot tool. In the Dashboard app you have the option to export the Dashboard as PDF. Right now it reports a timeout error because it is missing this attribute. Try to add following:

```
 <div
              data-shared-item="" // TODO: Remove data-shared-item and data-rendering-count as part of https://github.com/elastic/kibana/issues/179376
            >
              <StatusOverviewComponent />
            </div>
```

This manual adding of the data-shared-item attribute in each embeddable is temporary and should be holistically addressed later on in this [issue](https://github.com/elastic/kibana/issues/179376).

You can find more information about this [here](https://github.com/elastic/kibana/pull/181392)